src/widgetastic_patternfly/__init__.py

```diff
@@ -246,10 +246,17 @@ class FlashMessages(Widget):
     def __init__(self, parent, locator, logger=None):
         Widget.__init__(self, parent, logger=logger)
         self.locator = locator
+        self.MSG_XPATH = ('.//div[@id="flash_text_div" or '
```

Comment:
I don't think this needs to be defined in the constructor - it can just be a class attribute.

Design wise - that allows for easy modification by sub-classes for non-reference PF implementations.

Reply:
Second thought - This is really a property of the individual messages.

`FlashMessages` class could have a `message_widget_class` attribute, and we can reference `self.message_widget_class.LOCATOR` where needed.

That way the locator for the individual message is defined with the individual message class, and its still straightforward to subclass and replace the locator, or the widget class.

Reply:
Re-done to use ParametrizedView for each FlashMessage, indexed by its order in the FlashMessages block (updated PR description).